tests/specs/v2/rumXhrStatusTracking.js

```diff
@@ -42,75 +52,71 @@ describe("RUM status code tracking", function() {
       var pairStatus = expectedPairs[i].statusCode;
       var pairType = expectedPairs[i].type;
 
-      expect(payloadUrl).toBe(pairUrl, "failed for type: " + pairType);
-      expect(payloadStatus).toBe(pairStatus, "failed for type: " + pairType);
-      expect(payloadDataType).toBe(payloadDataType, "XHR data type missing for: " + pairType);
+      expect(payloadUrl === pairUrl).toBeTrue("failed for type: " + pairType);
+      expect(payloadStatus === pairStatus).toBeTrue( "failed for type: " + pairType);
+      expect(payloadDataType === payloadDataType).toBeTrue( "XHR data type missing for: " + pairType);
```

Comment:
we have lost the jasmine comparitors here?

Reply:
Yes. It throws a 'Negation comparison not an option' (or some thing like that).